front/public/locales/en/stdcm.json

```diff
@@ -5,11 +5,13 @@
     "consist": "Consist",
     "tractionEngine": "Traction engine"
   },
+  "formattedCreationDate": "{{month}}/{{day}}/{{year}} {{hours}}:{{minutes}}",
```

Comment:
Hm, I wonder if date format strings should really be in translation files: they're not translations. Can't we use e.g. `dateTimeFormatting` from `utils/date` here?